specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -5934,25 +5957,6 @@
       "description": "Filter for the Event Channel.",
       "type": "object",
       "properties": {
-        "subjectBeginsWith": {
-          "description": "An optional string to filter events for an event channel based on a resource path prefix.\r\nThe format of this depends on the publisher of the events. Wildcard characters are not supported in this path.",
-          "type": "string"
-        },
-        "subjectEndsWith": {
-          "description": "An optional string to filter events for an event channel based on a resource path suffix.\r\nWildcard characters are not supported in this path.",
-          "type": "string"
-        },
-        "includedEventTypes": {
-          "description": "A list of applicable event types that need to be part of the event channel. If it is desired to subscribe to all default event types, set the IncludedEventTypes to null.",
-          "type": "array",
-          "items": {
-            "type": "string"
-          }
-        },
-        "isSubjectCaseSensitive": {
-          "description": "Specifies if the SubjectBeginsWith and SubjectEndsWith properties of the filter\r\nshould be compared in a case sensitive manner.",
-          "type": "boolean"
-        },
```

Comment:
This is breaking change. Please share details on how it will not impact any customer.

Reply:
We had dicussed this in IM and email.